igor-web/src/main/groovy/com/netflix/spinnaker/igor/history/model/GenericBuildEvent.groovy

```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2016 Schibsted ASA.
+ * Copyright 2016 Schibsted, Inc.
```

Comment:
I think our legal name is actually `Schibsted ASA`, not `Schibsted, Inc`, as that name implies that the company is registered in the U.S., right?

Reply:
Oops, yeah - I buggered things - I'll fix. :)